ea_restaurant_data_loader/test/fixtures/app_client_fixture.ex

```diff
@@ -0,0 +1,28 @@
+defmodule EaRestaurantDataLoader.AppClientFixture do
+  alias EaRestaurantDataLoader.AppClient
+  alias EaRestaurantDataLoader.Repo
+
+  def build_app_client(client_name, client_id, user),
```

Comment:
Add user pattern match to be a User schema

Reply:
Done.